heartbeat/dovecot

```diff
@@ -0,0 +1,335 @@
+#!/bin/sh
+#
+# Resource script for Dovecot
+#
+# Description:  Manages Dovecot as an OCF resource in
+#               an high-availability setup.
+#
+# Author:       Raoul Bhatia <r.bhatia@ipax.at> : Original Author
+# License:      GNU General Public License (GPL)
+#
+#
+#       usage: $0 {start|stop|reload|monitor|validate-all|meta-data}
+#
+#       The "start" arg starts a Dovecot instance
+#
+#       The "stop" arg stops it.
+#
+# OCF parameters:
+#  OCF_RESKEY_binary
+#  OCF_RESKEY_config_file
+#  OCF_RESKEY_parameters
+#
+##########################################################################
+
+# Initialization:
+
+: ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
+. ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+
+# Defaults
+OCF_RESKEY_binary_default="/usr/sbin/dovecot"
+
+: ${OCF_RESKEY_binary=${OCF_RESKEY_binary_default}}
+: ${OCF_RESKEY_config_file=""}
+: ${OCF_RESKEY_pid_file=""}
```

Comment:
It seems like you should also use a `_default` variable for thisone, as it's empty by default, while the metadata shows a path it wont use by default.